emloop/main_loop.py

```diff
@@ -250,8 +275,7 @@ def _run_zeroth_epoch(self, streams: Iterable[str]) -> None:
 
         self._epoch_profile.clear()
         for stream_name in streams:
-            with self.get_stream(stream_name) as stream:
-                self.evaluate_stream(stream)
+            self.evaluate(stream_name)
```

Comment:
whoa, now you do not call `__enter__` and `__exit__` on the `StreamWrapper`, this may break the buffering I believe.

Reply:
I do, look at lines: https://github.com/iterait/emloop/pull/58/files/ce25ab0a86dbcf34cc7b44f936a3fd03f5fa9caa#diff-83cf5394877642e7670e44b807f8858bR228 and 
https://github.com/iterait/emloop/pull/58/files/ce25ab0a86dbcf34cc7b44f936a3fd03f5fa9caa#diff-83cf5394877642e7670e44b807f8858bR240.

Reply:
my bad